src/macros.rs

```diff
@@ -421,3 +421,56 @@ macro_rules! rawdoc {
         object
     }};
 }
+
+/// Same as [`serde_with::serde_conv!`], but with support for rustdoc documentation for the struct.
+#[macro_export]
```

Comment:
The rules around macros are weird but since this is intended as an internal-only macro I don't think we want `#[macro_export]` - that'll make it part of the public API of the crate.  Instead, you can add `pub(crate) use my_serde_conv;` after the `macro_rules!` definition to bring it into the normal namespace.